src/styles/main.scss

```diff
@@ -1,3 +1,239 @@
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+  }
+
+  &__navLink:hover {
+    color: $colorLink;
+  }
+}
+
+.is-active {
+  color: $colorLink;
+}
+
+.is-active::after {
+  content: " ";
+  display: block;
+  height: 4px;
+  width: 100%;
+  background-color: $colorLink;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+}
+
+.products {
+  display: flex;
+  flex-wrap: wrap;
+  flex-shrink: 0;
+  justify-content: center;
+  margin: 0 auto;
+  width: 1024px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  margin-right: 48px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__picture {
+    padding: 32px 19px 36px;
+  }
+  
+  &__images {
+    height: 134px;
+    width: 160px;
+  }
+  
+  &__name {
+    padding: 0 16px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+  
+  &__code {
+    color: #616070;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    padding: 4px 16px 0;
+  }
+  
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 16px 16px 0;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__prices {
+    display: flex;
+    flex-wrap: nowrap;
+    padding: 24px 16px 0;
+  }
+  
+  &__price:first-child {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    padding-right: 86px;
+    color: #616070;
+  }
+  
+  &__price:last-child {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
+
+  &__link {
+    margin: 16px;
+    text-align: center;
+    font-family: "Roboto", sans-serif;
+    display: block;
+    font-size: 0.1px;
+    font-weight: 700;
+    line-height: 40px;
+    color: #fff;
+    text-decoration: none;
+    text-transform: uppercase;
+    background: $colorLink;
+    border-radius: 5px;
+    position: relative;
+  }
+
+  &__link::before {
+    content: attr(data-name);
+    position: absolute;
+    font-size: 14px;
+    left: 0;
+    top: 0;
+    width: 100%;
+    color: $colorLink;
+    line-height: 38px;
+    background-color: #fff;
+    border: 1px solid $colorLink;
+    border-radius: 5px;
+    transform: rotateX(90deg) translateY(40px);
+    transform-origin: 50% 100%;
+    transition-duration: 300ms;
+    transition-property: transform;
+  }
+  
+  &__link:hover::before {
+    transform: rotateX(0deg)translateY(0);
+  }
+  
+  &__link::after {
+    content: attr(data-name);
+    position: absolute;
+    font-size: 14px;
+    left: 0;
+    top: 0;
+    width: 100%;
+    color: #fff;
+    line-height: 38px;
+    background-color: $colorLink;
+    border-radius: 5px;
+    transform: rotateX(0deg) translateY();
+    transform-origin: 50% 0;
+    transition-duration: 300ms;
+    transition-property: transform;
+  }
+  
+  &__link:hover::after {
+    transform: rotateX(90deg)translateY(-38px);
+  }
+
+}
+
+.card:nth-child(8) {
```

Comment:
why is it only applies for each 8 element? 
Isn't it better to use :not(:last-child) or something?